modular_skyrat/modules/blueshield/code/modsuit/mod_type.dm

```diff
@@ -3,13 +3,13 @@
 	icon = 'modular_skyrat/modules/blueshield/icons/praetorian.dmi'
 	icon_state = "praetorian-control"
 	theme = /datum/mod_theme/blueshield
-	applied_cell = /obj/item/stock_parts/cell/super
+	applied_cell = /obj/item/stock_parts/cell/high // Bubberstation Edit - Original: Super Cell
```

Comment:
do they really need that?

Reply:
High is 10 MJ. Super is 20 MJ. They can have a stock cell